ofctrl/ofSwitch.go

```diff
@@ -165,40 +149,43 @@ func (self *OFSwitch) IsReady() bool {
 }
 
 // Handle switch connected event
-func (self *OFSwitch) switchConnected() {
-	self.changeStatus(true)
-
-	// Send new feature request
-	self.Send(openflow13.NewFeaturesRequest())
-
-	self.Send(openflow13.NewEchoRequest())
```

Comment:
Echo request is sent periodically in function `switchConnected` in another goroutine.